system-int-test/test/cmr/system_int_test/search/granule_search_feature_id_crid_id_test.clj

```diff
@@ -0,0 +1,102 @@
+        coll2-concept-id (:concept-id coll2)
+        gran1 (d/ingest "PROV1"
+                        (dg/granule-with-umm-spec-collection
+                         coll1 coll1-concept-id {:day-night "DAY"
+                                                 :feature-ids ["feature1"]
+                                                 :crid-ids ["crid1"]})
+                        {:format :umm-json})
+        gran2 (d/ingest "PROV1"
+                        (dg/granule-with-umm-spec-collection
+                         coll1 coll1-concept-id {:day-night "DAY"
+                                                 :feature-ids ["FEATURE2"]
+                                                 :crid-ids ["CRID2"]})
+                        {:format :umm-json})
+        gran3 (d/ingest "PROV1"
+                        (dg/granule-with-umm-spec-collection
+                         coll1 coll1-concept-id {:day-night "DAY"
+                                                 :feature-ids ["super_power"]
+                                                 :crid-ids ["super_crid"]})
+                        {:format :umm-json})
+        gran4 (d/ingest "PROV2"
+                        (dg/granule-with-umm-spec-collection
+                         coll2 coll2-concept-id {:day-night "DAY"
+                                                 :feature-ids ["feature1" "feature2"]
+                                                 :crid-ids ["crid1" "crid2"]})
+                        {:format :umm-json})
+        gran5 (d/ingest "PROV2"
+                        (dg/granule-with-umm-spec-collection
+                         coll2 coll2-concept-id {:day-night "DAY"
+                                                 :feature-ids ["feature123"]
+                                                 :crid-ids ["crid123"]})
+                        {:format :umm-json})
+        gran6 (d/ingest "PROV2" (dg/granule-with-umm-spec-collection coll2 coll2-concept-id))]
+    (index/wait-until-indexed)
+
+    (testing "search by feature-id"
+      (are [items feature-ids options]
+        (let [params (merge {:feature-id feature-ids}
+                            (when options
+                              {"options[feature-id]" options}))]
+          (d/refs-match? items (search/find-refs :granule params)))
+
+        [] ["FAKE"] {}
+        [gran5] ["feature123"] {}
+        [gran1 gran4] ["feature1"] {}
+        [gran2 gran4] ["feature2"] {}
+        [gran1 gran2 gran4] ["feature1" "feature2"] {}
+
+        ;; ignore case
+        [gran2 gran4] ["FEATURE2"] {}
+        [gran2 gran4] ["FEATURE2"] {:ignore-case true}
+        [gran2] ["FEATURE2"] {:ignore-case false}
+
+        ;; pattern
+        [gran1 gran2 gran4 gran5] "feature*" {:pattern true}
+        [gran1 gran2 gran4] "feature?" {:pattern true}
+
+        ;; and option
+        [gran4] ["feature1" "feature2"] {:and true}))
+
+    (testing "search by crid-id"
+      (are [items crid-ids options]
```

Comment:
We shouldn't write new code using `are`, we should use `are3`.

Reply:
Not getting that memo. :) Felt `are3` would be too verbose in this case. I can change to use `are3` if there is another downvote on the `are` here.

Reply:
`are3` is more maintainable than `are` but only marginally.  During my time at CMR I've had instances where I was asked to convert tests I didn't write initially, as well as tests that I had initially written with `are`.  I don't care too much either way, I'm just passing along feedback that I've been given in the past.

Reply:
I agree that `are3` are better in most cases and it gives out the exact test case when the test fails which is really good. But in this case the tests are so trivial that it would make it actually more unreadable to use `are3`. No comments from other people, so I am going to merge it now.